ooniprobe/Test/Test/WebConnectivity.mm

```diff
@@ -18,6 +18,7 @@ -(void) runTest {
         [TestUtility downloadUrls:^(NSArray *urls) {
             [self setUrls:urls];
             [self setDefaultMaxRuntime];
+            [[NSNotificationCenter defaultCenter] postNotificationName:@"updateRuntime" object:nil];
```

Comment:
Do I understand correctly that here you're sending a message to the UI thread to draw the runtime?

Reply:
I tell the UI that something has happened (finished downloading url list) so the runtime should be updated. In this case written more than updated